foca/api/register_openapi.py

```diff
@@ -88,3 +97,24 @@ def register_openapi(
         logger.info(f"API endpoints specified in '{spec.path_out}' added.")
 
     return app
+
+
+def _mergeSpecs(original, *args, **kwargs):
+    # first element of the list is the main file
+    original_data = ConfigParser.parse_yaml(original)
+
+    for arg in args:
+        with open(arg):
+            to_merge = ConfigParser.parse_yaml(arg)
+        for key1 in to_merge:
```

Comment:
I'm actually not sure how generally applicable this is: why do you update exactly on level 2, why not on level 1, why not on level 3, 4, 5... ? What are the consequences and limitations of this approach? You could have a look at the `addict` package to update dictionaries smartly. Or use a package to merge json/yaml before parsing it with `ConfigParser.parse_yaml()`.